src/toil_scripts/lib/__init__.py

```diff
@@ -1,4 +1,3 @@
-# FIXME: replace with bd2k.util.iterables.flatten
 def flatten(x):
```

Comment:
BPL's flatten is more general, you should be able to do this:

```
from bd2k.util.iterables import flatten as bpl_flatten

def flatten(x):
    return list(bpl_flatten(x))
```

It will be slightly more efficient than the current implementation since only one list is being created.

You can leave the test case in-place.